src/main/java/com/google/sps/model/TasksClientImpl.java

```diff
@@ -61,6 +62,17 @@ private TasksClientImpl(Credential credential) {
     return taskLists != null ? taskLists : new ArrayList<>();
   }
 
+  @Override
+  public List<TaskList> listTaskLists(List<String> taskListTitles) throws IOException {
+    // returns null if no taskLists exist. Convert to empty list for ease.
+    List<TaskList> taskLists = tasksService.tasklists().list().execute().getItems();
+    taskLists = taskLists != null ? taskLists : new ArrayList<>();
+    taskLists.stream()
```

Comment:
This filtering logic should likely be tested (and as of now it cannot be tested) - I would actually suggest not making an overload for this method and instead using the more general `listTaskLists()` method, then creating a static helper method to do this filtering

Reply:
That's fair - after addressing the comments below, this PR is now more of a Locations Utility PR so I will definitely keep that in mind when polishing up the Go servlet in another PR. Thanks!

Reply:
`RESOLVED`